### PR TITLE
Add a note about Shadowrocket not being available in the China App Store

### PR DESCRIPTION
**Pull request by pjrobertson**
See #856 for more details.

Note: this isn't a fix for the project, just a suggestion/notice to inform users.

### CODE REVIEW

**Comment by pjrobertson** [comment]
Also note: my French isn't great, I've added in a French sentence here just so that the instructions in both languages stay in sync. Hopefully a native speaker will come along and improve it.

File: playbooks/roles/shadowsocks/templates/instructions.md.j2 [hunk 1]
```diff
@@ -106,7 +106,8 @@ This should return a 301 Found response **not** a connection refused error.
 
 <a name="ios"></a>
 ### iOS ###
-1. Download [Shadowrocket](https://itunes.apple.com/us/app/shadowrocket/id932747118?mt=8) and launch it.
+1. Download [Shadowrocket](https://itunes.apple.com/us/app/shadowrocket/id932747118?mt=8) and launch it.    
+> **Note:** Shadowrocket is not available in the China App Store. Apple was forced to remove it due to Chinese Government regulations. If you are using the China App Store and cannot find Shadowrocket, try switching to another country's App Store first.
```
Comment (cpu) [inline playbooks/roles/shadowsocks/templates/instructions.md.j2]: I think it's probably sufficient to say it's not available in China's app store and leave it at that. Thoughts?

Comment (alimakki) [inline playbooks/roles/shadowsocks/templates/instructions.md.j2]: @cpu +1

Comment (pjrobertson) [inline playbooks/roles/shadowsocks/templates/instructions.md.j2]: Hmm... I think it's important to let people know that there *is* a workaround by just switching your App store to another country.

If you guys prefer the shorter than that's cool, can you go ahead and edit my PR?

Comment (cpu) [inline playbooks/roles/shadowsocks/templates/instructions.md.j2]: I'm not familiar enough with the ecosystem to know how easy it is to change your app store country. I suspect that doing that can cause other problems and I'm hesitant to recommend that as a solution without more understanding of the steps required & potential side effects. 

> If you guys prefer the shorter than that's cool, can you go ahead and edit my PR?

Ok

Comment (cpu) [inline playbooks/roles/shadowsocks/templates/instructions.md.j2]: Updated with ae9c16b. @alimakki Can you +1?

File: playbooks/roles/shadowsocks/templates/instructions-fr.md.j2 [hunk 2]
```diff
@@ -105,7 +105,7 @@ Une fois que vous avez Shadowsocks fonctionnant localement, vous devrez transfé
 
 1. Téléchargez [Shadowrocket](https://itunes.apple.com/fr/app/shadowrocket/id932747118?mt=8) et lancez-le.
    * Vous serez invité à vous connecter à l'iTunes Store la première fois que vous exécutez Shadowrocket.     
-   > **Note:** Il n'y a pas Shadowrocket dans l'Itunes Store Chinoise, à cause des régulations du gouvernement Chine. Si vous ne trouve pas Shadowrocket dans votre Itunes Store, utiliser l'Itunes Store dans un autre pays.
+   > **Note:** Il n'y a pas Shadowrocket dans l'Itunes Store Chinoise.
```
Comment (alimakki) [inline playbooks/roles/shadowsocks/templates/instructions-fr.md.j2]: Il n'y a pas Shadowrocket dans l'Itunes Store Chinoise. -> Shadowrocket n'est pas disponible dans l'App Store chinois

Comment (cpu) [inline playbooks/roles/shadowsocks/templates/instructions-fr.md.j2]: Updated in 3c843ba. Thanks!

**Comment by cpu** [comment]
CI is all fscked up. Going to optimistically hope it fixes itself. Looks like an apt mirror thing. :-1:

**Pull request merged**